streaming-events.cabal

```diff
@@ -1,8 +1,9 @@
 cabal-version:      2.2
 name:               streaming-events
 version:            1.0.1
+category:           Web
 synopsis:           Client-side consumption of a ServerEvent.
-description:        Client-side consumption of a ServerEvent.
+description:        Client-side consumption of a ServerEvent, using streaming library.
```

Comment:
`cabal check` was complaining that description must be longer than synopsis, so I made up this bit to make it longer.